### PR TITLE
[multiple-choice]: add support for horizontal layout on settings

### PR DESCRIPTION
**Pull request by dianagociu**
Add support for horizontal/vertical layout toggle

image

### CODE REVIEW

**Comment by alextkd** [comment]
I fixed the snapshot and test passes fine now, I'm not sure why is not running. @andreeapescar / @elodszopos ?

**Comment by alextkd** [comment]
@dianagociu I would also reset the spaces updates

File: packages/multiple-choice/configure/src/defaults.js [hunk 1]
```diff
@@ -16,9 +16,14 @@ export default {
     rationaleEnabled: true,
     limitChoicesNumber: true,
     teacherInstructionsEnabled: true,
-    studentInstructionsEnabled: true
+    studentInstructionsEnabled: true,
+    verticalMode: true
   },
   configuration: {
+    verticalMode: {
+      settings: true,
```
Comment (andreeapescar) [inline packages/multiple-choice/configure/src/defaults.js:24]: I would rather not display the toggle for Vertical Mode in the Settings Panel (should be `settings: false`).
By default, in the model, it should be true, that's correct.

Comment (andreeapescar) [reply packages/multiple-choice/configure/src/defaults.js:24]: Also, we should add the new property in the docs. (pie-models)

File: packages/pie-models/src/pie/multiple-choice/index.ts [hunk 2]
```diff
@@ -32,6 +32,11 @@ export interface MultipleChoicePie extends PieModel {
   /** Determines if prompt should show */
   promptEnabled?: boolean;
 
+  /** Indicates the layout of choices for player
+   * @default: true
+   */
+  verticalLayout?: boolean;
```
Comment (andreeapescar) [inline packages/pie-models/src/pie/multiple-choice/index.ts:38]: I see you added **verticalMode** property, but in docs you've named it **verticalLayout**.

@dianagociu , can we please use verticalMode in the docs as well?

**Review by andreeapescar — APPROVED** [review]

**Pull request closed without merging**